src/plugins/ui_actions/public/cell_actions/components/cell_actions.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { orderBy } from 'lodash/fp';
+import React, { useCallback, useContext, useMemo, useRef } from 'react';
+import type { ActionExecutionContext } from '../../actions';
+import { CellActionsContext } from './cell_actions_context';
+import { InlineActions } from './inline_actions';
+import { HoverActionsPopover } from './hover_actions_popover';
+
+export interface CellActionConfig {
```

Comment:
I feel like some documentation here would be nice, especially as it's an important interface when interacting with the code for the first time